x-pack/plugin/spatial/src/test/java/org/elasticsearch/xpack/spatial/index/mapper/GeoShapeWithDocValuesFieldMapperTests.java

```diff
@@ -305,7 +301,10 @@ public void testSerializeDocValues() throws IOException {
 
     public void testShapeArrayParsing() throws Exception {
 
-        DocumentMapper mapper = createDocumentMapper(fieldMapping(this::minimalMapping));
+        DocumentMapper mapper = createDocumentMapper(fieldMapping(b -> {
```

Comment:
Should there be more tests in this class? Seems like we have added the most import capabilities to GeoShapeWithDocValuesFieldMapper, so I kind-of expected this test class to test more. Perhaps it is not needed because we have more tests elsewhere?